python/rpdk/python/resolver.py

```diff
@@ -33,3 +33,11 @@ def translate_type(resolved_type):
         return f"AbstractSet[{item_type}]"
 
     raise ValueError(f"Unknown container type {resolved_type.container}")
+
+
+def contains_model(resolved_type):
+    if resolved_type.container == ContainerType.LIST:
```

Comment:
Do we need something for `Set` or `Map`? Or are we OK treating non-list as Object?

Reply:
I might be missing something, but can't see a case where we ever end up with those, seeing as jsonschema only has `object` and `array` which we set to `dict` and `list`.

Reply:
on a deeper look I see that setting `"insertionOrder": false` or `"uniqueItems": true` will result in a set. https://github.com/aws-cloudformation/cloudformation-cli/blob/master/src/rpdk/core/jsonutils/resolver.py#L110-L117